mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java

```diff
@@ -151,6 +157,25 @@ private void initiateCenterCreation(CenterPayload centerPayload) {
 
         if (isCenterNameValid()) {
             mCreateNewCenterPresenter.createCenter(centerPayload);
+            AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(getActivity());
+            alertDialogBuilder
+                    .setMessage("Would you like to create another center?")
+                    .setPositiveButton("Yes", new DialogInterface.OnClickListener() {
+                        @Override
+                        public void onClick(DialogInterface dialogInterface, int i) {
+                            et_centerName.setText("");
```

Comment:
Please follow the coding style conventions. 
If the files you are working on don't follow the coding style conventions, considering changing them conforming to the rules stated at https://github.com/openMF/android-client/wiki/Code-style-conventions